extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -118,14 +122,16 @@ function ($, mw, loader, nirvana, tracker, labeling) {
 		modalInstance.bind('done', function runSave(e) {
 			processSave(modalInstance);
 
-			// Track - primary-button click
+			// Track - primary-button click or save by pressing Enter key
+			var label = e ? $(e.currentTarget).text() : 'keypress';
```

Comment:
Variable declarations should be at the top of the scope context [per conventions](https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#variables).